src/NuGet.Jobs.Catalog2Registration/Schema/JsonLdConstants.cs

```diff
@@ -65,6 +65,11 @@ public static class JsonLdConstants
                 Container = "@set",
                 Id = "tag",
             },
+            Vulnerabilities = new ContextTypeDescription
```

Comment:
I don't think we need this at all.

Reply:
@joelverhagen np--it does add the vulnerabilities entry to the json context though:
```
    ""vulnerabilities"": {
      ""@id"": ""vulnerability"",
      ""@container"": ""@set""
    }
```
and without it we don't see that there. I put it in for consistency with other collections like `tags`, but I'll remove it as you've mentioned a de-emphasis on JSON-LD. Is this inconsistency a problem though? I can make a separate change if you want it back in.